### PR TITLE
Version 1.0.2

### PR DESCRIPTION
**Pull request by carltongibson**
#576 Allows `django_filters` to be imported into settings

### CODE REVIEW

File: docs/guide/migration.txt [hunk 1]
```diff
@@ -8,6 +8,29 @@ instructions on how to migrate to the 1.0 release. A forwards-compatible 0.15
 release has also been created to help with migration. It is compatible with
 both the existing and new APIs and will raise warnings for deprecated behavior.
 
+Enabling warnings
+-----------------
+
+To view the deprecations, you may need to enable warnings within python. This
+can be avhieved with either the ``-W`` `flag`__, or with  ``PYTHONWARNINGS``
```
Comment (rpkilby) [inline docs/guide/migration.txt]: bleh

avhieved > achieved

Comment (carltongibson) [inline docs/guide/migration.txt]: https://github.com/carltongibson/django-filter/pull/577/commits/243b82281f7693c72e7580c86c3477c723aaf03a  👍

**Comment by askoretskiy** [comment]
@carltongibson Do you have any idea when it would be merged?

**Comment by carltongibson** [comment]
I'm kind of waiting for the 1.11RC.

**Comment by auvipy** [comment]
can we have a release based on django 1.11b1? rc1 is due march and we can release 1.0.3 after that?

**Comment by carltongibson** [comment]
@auvipy Main blocker now if just doing the change notes. If someone jumps in with those ([from the Milestone](https://github.com/carltongibson/django-filter/milestone/12)) I'll happily push the release. Otherwise I'll get to it ASAP but I had the rc-date in mind. 

(Installing from GitHub is an option — that's kind of why I've had the release PR open all this time...)

**Comment by pypetey** [comment]
django 1.11rc1 was released

**Comment by carltongibson** [comment]
@pypetey Yes, yes. I know. 😄 Coming later today. (All you people running the on the bleeding edge...)

**Comment by carltongibson** [comment]
https://github.com/carltongibson/django-filter/releases/tag/1.0.2

https://pypi.python.org/pypi/django-filter/1.0.2

Enjoy.

**Pull request merged**